iceoryx2-bb/lock-free/src/mpmc/container.rs

```diff
@@ -64,6 +64,11 @@ use std::{
     sync::atomic::{AtomicBool, Ordering},
 };
 
+/// A handle that corresponds to an element inside the [`Container`]. Will be acquired when using
+/// [`Container::add_with_handle()`] and can be released with [`Container::remove_with_handle()`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct ContainerHandle(u32);
```

Comment:
Food for thought, not sure if this is over the top
```rs
pub struct ContainerHandle{index: u32, container_id: u64}
```

Reply:
I was too quick and already merged it. But this is a good idea, I will add this to #111